<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QWidget" name="Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>520</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>759</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>759</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Main</string>
  </property>
  <widget class="QLabel" name="main_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>751</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gotham Narrow</family>
     <pointsize>24</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>PDF Araçları</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_merge">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>151</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gotham Narrow</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PDF Birleştir</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_split">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>100</y>
     <width>151</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gotham Narrow</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PDF Böl</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_convert">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>100</y>
     <width>151</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gotham Narrow</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PDF Dönüştür</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_filigran">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>151</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gotham Narrow</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PDF Filigran Ekle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_numarate">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>220</y>
     <width>151</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gotham Narrow</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PDF Numaralandır</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_edit">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>220</y>
     <width>151</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gotham Narrow</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PDF Düzenle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_communicate">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>400</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gotham Narrow</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>İstek, şikayet, iletişim</string>
   </property>
  </widget>
  <widget class="QLabel" name="owner_label">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>480</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Gotham Narrow</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>by mozgor19</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
